model/shared/src/main/scala/explore/model/ConfigurationForVisualization.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.data.NonEmptyList
+import cats.derived.*
+import cats.implicits.*
+import explore.model.syntax.all.*
+import lucuma.core.math.Angle
+import lucuma.core.math.Offset
+import lucuma.core.model.PosAngleConstraint
+import lucuma.schemas.model.BasicConfiguration
+
+// Yet another config class. This one is has the minmimal set of params to visualize the configuration
+// It is a subset of ObsConfiguration such that can be built out of either the db config
+// Or the minimal config from the modes table
+case class ConfigurationForVisualization(
```

Comment:
Have you considered moving all the constructors to the companion object here?

Eg: `fromItcInstrumentConfig` and `fromObsConfiguration`.

That way, those classes don't have to know about this one. Also you can make the default constructor `private`.

Reply:
Seems a good idea, I'll implement it